app/invocation/invocation_action_card.tsx

```diff
@@ -424,10 +424,8 @@ export default class InvocationActionCardComponent extends React.Component<Props
                       <div className="action-property-title">Execution metadata</div>
                       {this.state.actionResult.executionMetadata ? (
                         <div className="action-list">
-                          <div className="metadata-title">Worker</div>
+                          <div className="metadata-title">Execution Server Name</div>
```

Comment:
Should we call this the 'Executor Host ID'? 
My thinking:
We use 'executor' throughout our system to refer to the workers so it seems worthwhile to stay consistent. 
Host makes it clearer that we are referring to the machine that is hosting the executor rather than the executor itself.
And ID vs name because this value does not have any meaning to the user, it's ultimately a generated ID.

This is very subjective so please let me know if you disagree with any of this.

Reply:
+1 good name

Reply:
I like and will implement the name change, but on the topic of why I picked name, I do want to note that I think this ID could be something meaningful in the presence of a name override: https://github.com/buildbuddy-io/buildbuddy/blob/master/enterprise/server/remote_execution/executor/executor.go#L81-L84 , though that appears to be an option we don't currently use: https://github.com/buildbuddy-io/buildbuddy/blob/master/enterprise/server/cmd/executor/executor.go#L294

Reply:
I like the name change, but I do want to note that I think this ID could be something meaningful in the presence of a name override: https://github.com/buildbuddy-io/buildbuddy/blob/master/enterprise/server/remote_execution/executor/executor.go#L81-L84 , though that appears to be an option we don't currently use: https://github.com/buildbuddy-io/buildbuddy/blob/master/enterprise/server/cmd/executor/executor.go#L294

Reply:
Oh, actually, we call this entity an execution server extensively in the pre-existing code, should I change those names to `ExecutorHost` to be consistent with the new naming scheme? Otherwise I am concerned this could be confusing later.

Reply:
Talked offline; that would be work for a future PR, if ever.